### DISCUSSION TITLE
Can I run multiple apps with a single event loop

### DISCUSSION THREAD

**Discussion by luxapana** [uNetworking/uWebSockets#1309] [Q&A]
Is it possible for two or more instances of SSLApp to share single thread to rum the event loop?

**Comment by ghost** [comment]
Yes. App::run is really just a wrapper for Loop::run. So you only need to call run on the last app

**Reply by ChristianReese** [reply]
Is it possible to add additional apps to the thread/loop after it starts running? Such as via a call to defer?

EDIT: I gave this a try on Windows and it would seem that as long as the first app runs the loop, other apps can indeed be constructed/added to the running loop via a call to defer.
